Stack/Opc.Ua.Core/Types/Encoders/JsonEncoder.cs

```diff
@@ -314,8 +314,6 @@ public int Close()
 
             m_writer.Flush();
             int length = (int)m_writer.BaseStream.Position;
-            m_writer.Dispose();
```

Comment:
note: this is a small breaking change in the way the JSON encoder closes a stream. Turned out that this is a bug once other implementations than the default `MemoryStream` are being used. `MemoryStream` allows to access the data with e.g. ToArray after the stream is disposed, but other implementations like `RecycleMemoryStream` and `ArraySegmentStream` do not. This change makes it feasible to seperate the `Close` and the `Dispose`.